apps/rewards/app/components/Panel/ViewReward.js

```diff
@@ -0,0 +1,258 @@
+                    <SafeLink
+                      href={`https://rinkeby.etherscan.io/address/${creator}`}
+                      target="_blank"
+                      title={creator}
+                    >
+                      {truncatedCreator}
+                    </SafeLink>
+                  </p>
+                </div>
+              </Creator>
+            </div>
+            <div>
+              <h2>
+                <Label>Status</Label>
+              </h2>
+              <div>
+                {
+                  (
+                    Date.now() > endDate
+                  ) ? (
+                      <div>
+                        <IconCheck /> Available
+                      </div>
+                    ) : (
+                      <div>
+                        <IconTime /> Pending
+                      </div>
+                    )
+                }
+              </div>
+            </div>
+          </SidePanelSplit>
+          {description && (
+            <Part>
+              <React.Fragment>
+                <h2>
+                  <Label>Description:</Label>
+                </h2>
+                <p>{this.renderDescription(description)}</p>
+              </React.Fragment>
+            </Part>
+          )}
+          <SidePanelSplit>
+            <div>
+              <h2>
+                <Label>Reference Asset</Label>
+              </h2>
+              <p>{referenceToken}</p>
+            </div>
+            <div>
+              <h2>
+                <Label>Type</Label>
+              </h2>
+              <div>
+                {isMerit?'Merit Reward':'Dividend Reward'}
+              </div>
+            </div>
+          </SidePanelSplit>
+          <SidePanelSplit>
+            <div>
+              <h2>
+                <Label>Amount</Label>
+              </h2>
+              <p>
+                {displayCurrency(amount)}{' '}{translateToken(rewardToken)}
+              </p>
+            </div>
+            <div>
+              <h2>
+                <Label>Period</Label>
+              </h2>
+              <div>
+                {Intl.DateTimeFormat().format(startDate)}
+                {' - '}
+                {Intl.DateTimeFormat().format(endDate)}
+              </div>
+            </div>
+          </SidePanelSplit>
+          <Part>
+            <Text size='large' weight='bold' >Reward Summary</Text>
+          </Part>
+          <Info>
+            <TokenIcon />
+            <Summary>
+              <p>
+                {'A total of '}
+                <SummaryVar>{displayCurrency(amount)} {translateToken(rewardToken)}</SummaryVar>
+                {' will be distributed as a reward to addresses that earned '}
+                <SummaryVar>{referenceToken}</SummaryVar>
+                {' from '}
+                <SummaryVar>
+                  {startDate.getDate().toString() + '-' + MONTHS[startDate.getMonth()] + '-' + startDate.getFullYear()}
```

Comment:
there is formatting utility in date-fns...

`import { format } from 'date-fns'`
`format(startDate, 'dd-MMM-yyyy')`